Documentation/installation/bgp-control-plane.rst

```diff
@@ -297,7 +297,7 @@ error is logged and the ``BGPRouterManager`` will continue to the next
 ``CiliumBGPVirtualRouter``, utilizing the aforementioned logic.
 
 GoBGP BGPRouterManager Architecture
-                                   
+
```

Comment:
This looks like there should be a title underline here?

The formatting on this whole section is messed up. All sentences are separated by empty lines. Would you mind reformatting the paragraphs and item list (above) correctly? I can do as a follow-up otherwise.

Reply:
Sure, added in a second commit.